sig-scalability/extending_slo.md

```diff
@@ -0,0 +1,71 @@
+# Extended Kubernetes scalability SLOs
```

Comment:
We certainly should think where to put it and how to advertise it. We also need to work on SIG meeting on filling up cluster configuration for tests. I'm thinking about putting there our sizing guides, to have anything to begin discussion.